app/Console/Commands/EnergiDataServiceLoadChargeGroups.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace App\Console\Commands;
+
+use App\Models\ChargeGroup;
+use App\Services\GetEnergiDataServiceChargeGroups;
+use Illuminate\Console\Command;
+
+class EnergiDataServiceLoadChargeGroups extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'energidata:request-and-store-charge-groups';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Requests data about grid areas from EnergiDataService. Subarea is called "Charge Groups" at EnergiDataService. All data is stored to local data storage';
+
+    private GetEnergiDataServiceChargeGroups $getDatahubChargeGroups;
+
+    /**
+     * Create a new command instance.
+     *
+     * @return void
+     */
+    public function __construct(GetEnergiDataServiceChargeGroups $datahubPriceLists)
```

Comment:
En uskreven regel .. aldrig brug construct in commands. Grunden til det er at hver gang Laravel booter - så kører den construct for alle commands. Det betyder at du initialisere GetEnergiDataServiceChargeGroups hver gang eksempelvis et endpoint bliver kaldt eller alt andet. Jeg har før set eksempler på at man i construct - rent faktisk udfører et eller andet - hvilket kan give noget frygteligt rod, som kan være svært at gennemskue 😄 

Så istedet for er det bedre at i `handle()`

```php
$getDatahubChargeGroups = app(GetEnergiDataServiceChargeGroups::class);
```

Reply:
Det har jeg aldrig haft skænket en tanke. Det var da godt lige at vide :smiley:
Det kunne jeg sagtens forestille mig kunne skabe alverdens ulykker og at man så ville kunne komme til at bruge timer på at gennemskue, hvad f..... der foregår.
Det er jeg glad for lige at blive klogere på :smiley: